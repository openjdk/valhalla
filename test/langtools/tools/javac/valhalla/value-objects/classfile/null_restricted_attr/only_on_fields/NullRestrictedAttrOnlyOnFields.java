/* @test /nodynamiccopyright/
 * @summary [lw5] check that there can only be one NullRestricted attribute
 * @modules jdk.compiler/com.sun.tools.javac.api
 *          jdk.compiler/com.sun.tools.javac.main
 *          jdk.compiler/com.sun.tools.javac.util
 * @library /tools/lib
 * @compile NullRestrictedOnMethod.jcod
 * @compile/fail/ref=NullRestrictedAttrOnlyOnFields.out -XDrawDiagnostics NullRestrictedAttrOnlyOnFields.java
 */

public class NullRestrictedAttrOnlyOnFields {
    void m() {
        NullRestrictedOnMethod v = new NullRestrictedOnMethod();
    }
}
