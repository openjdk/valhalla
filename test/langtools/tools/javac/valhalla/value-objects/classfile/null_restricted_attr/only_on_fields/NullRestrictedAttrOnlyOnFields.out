NullRestrictedAttrOnlyOnFields.java:13:9: compiler.err.cant.access: NullRestrictedOnMethod, (compiler.misc.bad.class.file.header: NullRestrictedOnMethod.class, (compiler.misc.attribute.only.applicable.to.fields: NullRestricted))
1 error
